gspread/models.py

```diff
@@ -563,6 +564,36 @@ def insert_row(self, values, index=1):
                 newcell.value = rows_after_insert[r - 1][c - 1]
                 updated_cell_list.append(newcell)
         self.update_cells(updated_cell_list)
+    
+
+    def insert_column(self, values, index=1):
+        self.add_cols(1)
+        data_width = len(values)
+        if self.row_count < data_width:
+            self.resize(rows=data_width)
+	
+	all_cells = self.get_all_values()
+        j=0
+	if not all_cells:
+	    columns_after_insert = [[]]
+	else:
+	    columns_before_insert = [[] for k in range(len(all_cells[0]))]
+            while j<len(all_cells[0]):
```

Comment:
I have corrected all the indentations. Please take a look into the latest commit.